jinja/carnival/attractions.html

```diff
@@ -81,25 +81,72 @@ <h5 class="mb-1 black-text px-3">Nov 12</h5>
-				      <div class="col">
-					      <i class="fas fa-laptop-code fa-5x" style="color:#47e4ca"></i>
-					<h3 class="card-title card-title">
-					    <strong>Coding Challenges</strong>
-					</h3>
-				      </div>
-				    </div>
-				   <h5 class="mb-1 black-text px-3">Nov 16-30</h5>
-				  </div>
-
-                        </div>
+          <div class="col-md-4 mb-5">
+                  <div class="card card-image" style="background-image: url(../img/ferris_wheel_blur.png); background-size: cover;">
+                  <a href="community_calls.html">
+          <div class="text-center d-flex align-items-center rgba-white-strong mask">
+
+                <div class="col-md-12">
+                  <div class="row my-4">
+                    <div class="col">
+                        <i class="fas fa-phone fa-5x" style="color:#474ae4"></i>
+                  <h3 class="card-title card-title">
+                      <strong>Community Calls</strong>
+                  </h3>
+                    </div>
+                  </div>
+                 <h5 class="mb-1 black-text px-3">Nov 4, 11, 18 and 25</h5>
+                </div>
+
+                      </div>
+        </a>
+              </div>
+          </div>
+         <div class="col-md-4 mb-5">
+                  <div class="card card-image" style="background-image: url(../img/fireworks_blur.png); background-size: cover;">
+                      <a href="prize_booth.html">
+          <div class="text-center d-flex align-items-center rgba-white-strong mask">
+
+                <div class="col-md-12">
+                  <div class="row my-4">
+                    <div class="col">
+                        <i class="fas fa-gift fa-5x" style="color:#ff1403"></i>
+                  <h3 class="card-title card-title">
+                      <strong>Prize Booth</strong>
+                  </h3>
                     </div>
+                  </div>
+                 <h5 class="mb-1 black-text px-3">Dec 15</h5>
+                </div>
+
+                      </div>
+        </a>
+                  </div>
+      
+          </div>
+          <div class="col-md-4 mb-5">
+            <div class="card card-image" style="background-image: url(../img/ferris_wheel_blur.png); background-size: cover;">
+                <div class="text-center d-flex align-items-center rgba-white-strong mask">
+                  <div class="col-md-12">
+                    <div class="row my-4">
+                      <div class="col">
+                        <i class="fas fa-laptop-code fa-5x" style="color:#47e4ca"></i>
+                  <h3 class="card-title card-title">
+                      <strong>Coding Challenges</strong>
+                  </h3>
+                      </div>
+                    </div>
+                   <h5 class="mb-1 black-text px-3">Nov 16-30</h5>
+                  </div>
+                </div>
             </div>
+    </div>
+          
+
+      </div>
+
+        <div class="row">
```

Comment:
```suggestion
        <div class="row justify-content-md-center">
```
This should make it look like this:
![image](https://user-images.githubusercontent.com/7213358/141362821-6b640466-a6c1-4333-9fcb-8f2ab34ec642.png)